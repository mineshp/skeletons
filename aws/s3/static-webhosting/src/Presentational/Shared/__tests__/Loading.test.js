import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';

describe('Presentational > Loading', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(
      <Loading />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
