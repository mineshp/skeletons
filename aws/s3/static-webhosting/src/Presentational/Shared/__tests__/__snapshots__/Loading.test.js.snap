// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentational > Loading Renders correctly 1`] = `
<div>
  <Dimmer
    active={true}
  >
    <Loader
      size="medium"
    >
      Magic is happening...
    </Loader>
  </Dimmer>
</div>
`;
